openlibrary/templates/account/create.html

```diff
@@ -12,8 +12,10 @@ <h1>$_("Sign Up")</h1>
 <div id="contentBody">
 
 <script type="text/javascript">
-\$().ready(validateForms);
-\$().ready(function(){
+window.q.push( function() {
+    validateForms();
+});
+window.q.push( function(){
```

Comment:
You can merge these into one single function push.
There's no reason to have more than one window.q.push call.

The window.q simply registers a callback which will run in a FIFO fashion when JS is loaded.